index.js

```diff
@@ -193,6 +196,21 @@ function onchange (match, command, rawArgs, opts = {}) {
   watcher.on('error', (error) => log(`watcher error: ${error}`))
 }
 
+function getIgnoreMergedFromIgnoreFile(exclude = [], ignorePath) {
```

Comment:
It looks like Prettier uses https://github.com/kaelzhang/node-ignore#addignorefilepath, can we do the same?

Reply:
I even studied the way prettier works with node-ignore but to implement how they are using we would have to bring much of the watch handling into the codebase. This module works with the idea of ​​path validation from a list of ignores. What we are doing is sending an array for chokidar to handle from it.

Reply:
Why don't you just give `chokidar` the function returned by ignore? It seems like there'd be no customization then. Otherwise I'd prefer you write tests to cover this functionality, it should be easier to test than file events.

Reply:
done